src/docs/backend/development-server.mdx

```diff
@@ -27,3 +27,31 @@ This will do literally nothing except bring up the web workers. You are now resp
 * Starting Celery workers using `sentry run worker` to run most kinds of background jobs.
 
 Use `sentry run --help` to see what you might be missing by not running devserver.
+
+
+## Using ngrok
+
+The development server can be operated behind ngrok for when you need to receive traffic from the outside world. This comes up frequently when working with integrations. Start off by defining a configuration file for `ngrok`:
+
+```yaml
+version: '2'
+authtoken: <YOUR-NGROK-AUTHTOKEN>
+tunnels:
+  acme-org
+    proto: http
+    hostname: $yourname.ngrok.dev
```

Comment:
nit: I'd use `<yourname>` or just `yourname`, because with `$` it kind of looks like you'd be able to use an env var, which won't actually work in a YAML.